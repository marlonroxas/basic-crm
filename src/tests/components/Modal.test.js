import React from 'react'
import { shallow } from 'enzyme'

import Modal from '../../components/Modal/index'

test('Should render Modal correctly', () => {
  const wrapper = shallow(<Modal />)
  expect(wrapper).toMatchSnapshot()
})