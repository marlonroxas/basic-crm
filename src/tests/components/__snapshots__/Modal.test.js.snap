// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Modal correctly 1`] = `
<div
  className="modal"
>
  <div
    className="modal-content"
  >
    <div
      className="modal-content__title"
    />
    <div
      className="modal-content__body"
    />
  </div>
</div>
`;
